charts/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: api7-ingress
+    control-plane: controller-manager
+  name: {{ .Release.Name }}-metrics-service
+  namespace: {{ .Values.namespace }}
+spec:
+  ports:
+  - name: https
+    port: 8443
+    protocol: TCP
+    targetPort: 8443
+  selector:
```

Comment:
please ref: https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix-ingress-controller/templates/_helpers.tpl

Labels and selector cannot be a fixed value.

Reply:
done